config/exercise_readme.go.tmpl

```diff
@@ -26,14 +26,15 @@ To run the tests, run the appropriate command below ([why they are different](ht
 - Python 3.3+: `pytest {{ .Spec.SnakeCaseName }}_test.py`
 
 Alternatively, you can tell Python to run the pytest module (allowing the same command to be used regardless of Python version):
-`python -m pytest {{ .Spec.SnakeCaseName }}_test.py`
+`python -m pytest {{ .Spec.SnakeCaseName }}_test.py`.
```

Comment:
You don't need a period. `python -m pytest {{ .Spec.SnakeCaseName }}_test.py` is complete.